frontend/lib/identification/verification_workflow/verification_qr_content_parser.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:convert';
+
+import 'package:ehrenamtskarte/identification/qr_code_scanner/qr_code_processor.dart';
+import 'package:ehrenamtskarte/proto/card.pb.dart';
+
+class VerificationParseException extends QrCodeParseException {
+  final String internalMessage;
+  final Exception cause;
+  final StackTrace? stackTrace;
+
+  VerificationParseException({required this.internalMessage, required this.cause, this.stackTrace})
+      : super(internalMessage);
+}
+
+DynamicVerifyCode parseQRCodeContent(String rawBase64Content) {
+  const base64Decoder = Base64Decoder();
+
+  // TODO (Max): Refactor into Dart extension
```

Comment:
This comment should go somewhere else no because this function no longer should access extensions.